models/sklearn/classification/classification_task_manager.py

```diff
@@ -0,0 +1,136 @@
+import os
+from time import time
+from termcolor import colored
+from utils import load_yaml, FindCreateDirectory, TrainingProcesses
+from classification.classification_task import ClassificationTask
+from datetime import datetime
+from logging_tool import LoggerSetup
+
+
+validClassifiers = ['NN', 'svm']
+validEvaluations = ['nfoldcrossvalidation']
+
+
+class ClassificationTaskManager:
+    """
+
+    """
+    def __init__(self, config, train_class, X, y, tracks, exports_path, log_level):
+        """
+
+        :param yaml_file: The configuration file name
+        :param train_class: The class that will be trained
+        :param X: The already shuffled data that contain the features
+        :param y: The already shuffled data that contain the labels
+        """
+        self.config = config
+        self.train_class = train_class
+        self.X = X
```

Comment:
I could change X and y to features and labels accordingly. I preferred to name these variables like that because of the standard terminology by sklearn and the Machine Learning in general. What do you think, should I change it? Check an example here: https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html